docs/sma/plugin-pdf.md

```diff
@@ -166,6 +166,7 @@ Non-hotkeys
 			- Add to selection (Hold ctrl)
 			- Select image (Click image)
 ```
+<p class="tip">⚠️ When you extract text from the PDF file, the windows error sound may play.</p>
```

Comment:
1) Syntax

Use markdown instead. HTML is reserved for these occasions where we are forced to use it (e.g. for something that is not possible with markdown)

The equivalent for this is:
```md
> [!NOTE]
> When you extract text from the PDF file, the windows error sound may play.
```

2) It is nice to warn the user, but it is even better to provide a solution. Fortunately, you already have one ! Inform the user and link to the FAQ.

Every heading (all the markdown lines that begin with # up to -> ######) has an "id" that acts as an anchor. You can check for the heading's ID by moving your mouse over it in your browser, and checking where it sends you to (the url that appears at the bottom).

Here you could add something like `A work around is to [disable the system sounds](#faq) for the PDF plugin.`

The `[xx](link)` format in markdown is used to create a link to a page, or part of a page. Here we are using only #faq. # means that we want to send to an anchor in the page, and this anchor is called faq.

```suggestion
> [!NOTE]
> When you extract text from the PDF file, the windows error sound may play.
> A work around is to [disable the system sounds](#faq) for the PDF plugin.
```